tests/datasets/test_views.py

```diff
@@ -82,91 +86,104 @@ def test_distribution_preview(app: DjangoTestApp, dataset_detail_data):
     assert resp.json == {'data': [['Column'], ['Value']]}
 
 
+def rebuild_index():
+    ui = UnifiedIndex()
+    index = DatasetIndex()
+    ui.build(indexes=[index])
+    haystack.connections["default"]._index = ui
+
+    backend = haystack.connections["default"].get_backend()
+    backend.clear()
+    backend.update(index, Dataset.objects.all())
```

Comment:
Use [pytest-django-haystack](https://pypi.org/project/pytest-django-haystack/).

Reply:
Could not add pytest-django-haystack, because it uses method getfuncargvalue, which is not available anymore. It was changed to getfixturevalue.  So I copied the code from  pytest-django-haystack and added haystack marker to conftest.py with changed method name.